shotover/benches/benches/codec/kafka.rs

```diff
@@ -0,0 +1,147 @@
+use bytes::{Bytes, BytesMut};
+use criterion::{black_box, criterion_group, BatchSize, Criterion};
+use shotover::codec::kafka::KafkaCodecBuilder;
+use shotover::codec::{CodecBuilder, Direction};
+use shotover::message::{Message, ProtocolType};
+use tokio_util::codec::{Decoder, Encoder};
+
+const KAFKA_REQUESTS: &[(&[u8], &str)] = &[
+    (
+        include_bytes!("kafka_requests/metadata.bin"),
+        "request_metadata",
+    ),
+    (
+        include_bytes!("kafka_requests/list_offsets.bin"),
+        "request_list_offsets",
+    ),
+    (include_bytes!("kafka_requests/fetch.bin"), "request_fetch"),
+    (
+        include_bytes!("kafka_requests/produce.bin"),
+        "request_produce",
+    ),
+];
+
+fn criterion_benchmark(c: &mut Criterion) {
+    let mut group = c.benchmark_group("kafka_codec");
```

Comment:
Since the group is now `kafka_codec` the bench names should have the `kafka_` prefix removed.
Otherwise the bench names have `kafka` twice which is redundant
![image](https://github.com/shotover/shotover-proxy/assets/5120858/08087189-f81a-4827-8d33-eb72d960183f)


Or alternatively we could just keep as `codec`.

Same thing for cassandra_codec.